docs/nodes/nodes-library/nodes/Slack/README.md

```diff
@@ -68,16 +69,96 @@ The final workflow should look like the following image.
 
 The Start node exists by default when you create a new workflow.
 
-### 2. Slack Node
+### 2. Slack node (create: channel)
 
-1. Click on the plus button, search for "Slack", select it and double-click on the new node.
-2. Select OAuth2 in the *Authentication* field and select "Create new" in the *Slack OAuth2 API* field.
-3. Create a credential as shown in [this tutorial](../../../credentials/Slack/).
-4. Enter the name of the channel in the *Channel* field.
-5. Enter the text of the message to be posted in *Text* field.
-6. Browse the *Attachments*, *Other Options*, and *Blocks* to see if you would like to use any of them.
-7. Click on *Execute Node* to run the workflow.
+This node will create a new channel in your Slack workspace. This operation requires the `channel:manage` scope. Add this scope under the ***Bot Token Scopes*** section on Slack.
```

Comment:
Is there a link to step by step instructions in the docs or maybe in Slack's docs?

Reply:
Couldn't find separate documentation that has step-by-step instructions. However, I found [this](https://api.slack.com/authentication/basics#scopes). This might confuse our reader though.

Reply:
I remember that we have some instructions for the reader in the credential docs for this. Does it make sense to point them there for that? If not, then we should probably add something in the FAQs.

Reply:
Add the instructions in the FAQs.